### DISCUSSION TITLE
Permissions denied when trying to clone a Llama2 13b model after being approved by Meta and following all instructions.

### DISCUSSION THREAD

**Discussion by 3dstudiodesign** [community/community#78343] [Programming Help] [answered]
First, I want to clarify that I received permissions from Meta to access the Llama2 model, and it displays on the Meta page for this model that I have permission.

So, then in git bash, I generated a SSH key pair successfully, added a passphrase, my identity was saved, public key was saved, and a key fingerprint was made, the key's random art image was provided.  

However when trying to complete the git clone, I typed in the git clone command for the ''meta-llama/Llama-2-13b-chat-hf'' model provided on HF and I received an error:

"The authenticity of host hf.co (34.198.14.237) can't be established.
<host> key fingerprint is <key fingerprint>. This key is not known by any other names. Are you sure you want to continue connecting (yes/no/[fingerprint])?"

So, I typed yes (the first time) and was denied with the below error:  

"Warning: Permanently added 'hf.co' (host) to the list of known hosts.
git 'at' hf.co: Permission denied (publickey).
fatal: Could not read from remote repository."

So, the second time, when I typed in the git clone command and was asked for yes/no/[fingerprint], I copied and pasted the [fingerprint] given and was again  denied with this error:

"Permission denied (publickey).
fatal: Could not read from remote repository."

The third time I typed in the git clone command, I was not given the option to type y/n/fingerprint, and received the message:

"Cloning into 'Llama-2-13b-chat-hf'...
git 'at' hf.co: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists."

I also tried the http option and received an error message that it was "deprecated"

So, I am not sure why it is not connecting to the Meta model that I have permissions to access.  Any suggestions?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @3dstudiodesign ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by 3dstudiodesign** [reply, marked as answer]
I aplogize.  This is the area I was guided to ask my question.
